src/common/etc/profile.d/xcp-ng-prompt.sh

```diff
@@ -4,7 +4,7 @@
 
 # BASH : Configure things differently if current terminal has >= 8 colors.
 if [ "$PS1" ] && [ $(tput colors 2>/dev/null) -ge 8 ]; then
-    PS1='[\[\e[1;38;5;23m\]\D{%H:%M} \[\e[38;5;167m\]\h \[\e[m\]\W]\$ '
+    PS1='\[\e[0;38;5;160m\][\[\e[0;2m\]\A \[\e[0;1;38;5;105m\]\H \[\e[0m\]\W\[\e[0;38;5;160m\]]\[\e[0m\]\$ \[\e[0m\]'
```

Comment:
Why did `\h` ("the hostname up to the first `.`") become `\H` (full hostname)? And it's not consistent with the color-less prompt in the `else` block.

And if we replace `\D{%H:%M}` with `\A` (I think they're equivalent), then we should also do it in the `else` block.

Reply:
Also, is the final `\[\e[0m\]` necessary? We already switch back to white before `\$`.

Reply:
I used a generator to do it: https://scriptim.github.io/bash-prompt-generator/

So I'm not sure about the details.